utils/collectd/files/exec-scripts/sqm_collectd.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+. /usr/share/libubox/jshn.sh
+
+HOSTNAME="${COLLECTD_HOSTNAME:-localhost}"
+INTERVAL="${COLLECTD_INTERVAL:-60.000}"
+
+handle_cake() {
+	local ifc ifr tin i
+
+	ifc="$1"
+	ifr="${ifc//./_}"
+
+#Overall
+	json_get_vars bytes packets drops backlog qlen
+
+#Options
+	json_select options
+	json_get_vars bandwidth
+	json_select ".."
+
+	echo "PUTVAL \"$HOSTNAME/sqm-$ifc/qdisc_bytes\" interval=$INTERVAL N:$bytes"
+	echo "PUTVAL \"$HOSTNAME/sqm-$ifc/qdisc_drops\" interval=$INTERVAL N:$drops"
+	echo "PUTVAL \"$HOSTNAME/sqm-$ifc/qdisc_backlog\" interval=$INTERVAL N:$backlog"
+
+#Tins
```

Comment:
Not a fan of the comments not matching indent...

Reply:
Done, I think.